src/renderer/shared/i18n/locales/en.json

```diff
@@ -1543,6 +1543,17 @@
       "staking": "Staking operations",
       "unknown": "Unknown operations"
     },
+    "types": {
+      "Any": "Any",
+      "Auction": "Auction",
+      "CancelProxy": "Cancel",
+      "Governance": "Governance",
+      "IdentityJudgement": "Judgement",
+      "NominationPools": "Nominations",
+      "NonTransfer": "Non Transfer",
+      "Staking": "Staking",
+      "Unknown": "Unknown"
```

Comment:
I think it is much more useful to show raw proxy type name in case we were not able to resolve corresponding translation